plugins/sam3dj.js

```diff
@@ -0,0 +1,343 @@
+/** sam3DJ | aka. Sammy's .3DJ (format implementation)
+ *  ==================================================
+ *  Author:     Sammy L. Koch (1Turtle)
+ *  Licence:    Apache-2.0 (see https://www.apache.org/licenses/LICENSE-2.0.txt)
+*/
+(function() {    
+    const pluginProperties = {
+        title: 'sam3DJ',
+        author: '1Turtle',
+        tags: ['Minecraft: Java Edition', 'format', '3dj'],
+        version: '1.0.0',
+        variant: 'both',
+        description: 'Export models to the .3DJ format!',
+        about: `This plugin allows you to **export a model into the .3DJ format**.  \n

```

Comment:
Please write this all in one line, like in plugins.json. The indentation confuses the markdown renderer, causing the text to be displayed as code.